server.js

```diff
@@ -865,7 +869,7 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
         try {
           var data = JSON.parse(buffer);
 
-          var value = data[0].msr[0].val;
+          var value =  parseInt((!!version && version < 5.4) ? data[0].msr[0].val : data.component.measures[0].value);
```

Comment:
Could you store this expression in a variable? It'll make this + above a bit more readable.

```
const useLegacyApi = !!version && version < 5.4;
```